test/Mono.Linker.Tests.Cases/RequiresCapability/RequiresCopyAssembly.cs

```diff
@@ -0,0 +1,66 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics.CodeAnalysis;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Mono.Linker.Tests.Cases.Expectations.Assertions;
+using Mono.Linker.Tests.Cases.Expectations.Metadata;
+using Mono.Linker.Tests.Cases.RequiresCapability.Dependencies;
+
+namespace Mono.Linker.Tests.Cases.RequiresCapability
+{
+	[SetupLinkerAction ("copy", "lib")]
+	[SetupCompileBefore ("lib.dll", new[] { "Dependencies/RequiresInCopyAssembly.cs" })]
+	[KeptAllTypesAndMembersInAssembly ("lib.dll")]
+	[SkipKeptItemsValidation]
+	// Annotated members on a copied assembly should not produce any warnings
+	// unless directly called or referenced through reflection.
+	[LogDoesNotContain ("--UncalledMethod--")]
+	[LogDoesNotContain ("--getter UnusedProperty--")]
+	[LogDoesNotContain ("--setter UnusedProperty--")]
+	[LogDoesNotContain ("--UnusedBaseTypeCctor--")]
+	[LogDoesNotContain ("--UnusedVirtualMethod1--")]
+	[LogDoesNotContain ("--UnusedVirtualMethod2--")]
+	[LogDoesNotContain ("--IUnusedInterface.UnusedMethod--")]
+	[LogDoesNotContain ("--UnusedImplementationClass.UnusedMethod--")]
+	// [LogDoesNotContain ("UnusedVirtualMethod2")] // https://github.com/dotnet/linker/issues/2106
+
+	[ExpectedNoWarnings]
+	class RequiresCopyAssembly
```

Comment:
Maybe `RequiresWithCopyAssembly` ... otherwise it sounds like the test requires a copy assembly to exist...